src-backend/api/views.py

```diff
@@ -37,6 +39,25 @@ def get_queryset(self):
         user = self.request.user
         return models.Page.objects.filter(procedure__owner_id__exact=user.id)
 
+    @list_route(methods=['PATCH'])
+    def partial_bulk_update(self, request, *args, **kwargs):
```

Comment:
I'd think that bulk update is sufficient. If a field is missing, then we can assume it's unchanged.